addons/mass_mailing/static/src/js/mass_mailing_widget.js

```diff
@@ -42,6 +42,9 @@ var MassMailingFieldHtml = FieldHtml.extend({
         var self = this;
         var fieldName = this.nodeOptions['inline-field'];
 
+        if (this.mode == "readonly") {
```

Comment:
the condition does not cover all cases. If you click on 'edit' then directly on 'save':

Uncaught TypeError: Cannot read property 'find' of undefined
http://451917-30533-f519b2.runbot15.odoo.com/mass_mailing/static/src/js/mass_mailing_widget.js:48

Reply:
Hello @Gorash ,
This PR is related to this bug https://www.odoo.com/web#id=1932061&action=333&active_id=1278&model=project.task&view_type=form&menu_id=4720

The case you are talking about will be taken care in another task with ref link https://www.odoo.com/web#id=1932687&action=333&active_id=131&model=project.task&view_type=form&menu_id=4720

Reference  PR https://github.com/odoo/odoo/pull/30833 

SO if you want to merge both into one let me know so we will merge both issue with the one pr and will cancel another one. 

Thanks

Reply:
:+1: